content/js/query.md

```diff
@@ -139,19 +143,23 @@ query.addAscending('age');    // sorted by age increasing order
 query.addDescending('price'); // sorted by price decreasing order
 ```
 
+The above query will first sort from small age to large age, and records
+with the same age will be then sorted from high price to low price.
```

Comment:
Uhmm, I'm not so familiar with SQL syntax 😂, so shall I just copy and paste `ORDER BY age, price DESC`?